### PR TITLE
Debugging, cleanup

### PR DESCRIPTION
**Pull request by jonsterling**

### CODE REVIEW

File: src/core/Restriction.ml [hunk 1]
```diff
@@ -5,12 +5,18 @@ type dim = I.t
 
 type eqn = dim * dim
 
-module UF = DisjointSet.Make (PersistentTable.M)
```
Comment (jonsterling) [inline src/core/Restriction.ml]: @favonia I don't know if the changes in this file are useful at all. should we revert or keep them?

**Comment by jonsterling** [comment]
btw don't worry about this PR, it's most garbage experimentation, but I will extract some things from it ultimately.

File: example/torus.red [hunk 2]
```diff
@@ -21,7 +21,7 @@ let t2c (t : torus) : s1 × s1 =
   | pt ⇒ <base, base>
   | p/one i ⇒ <loop i, base>
   | p/two i ⇒ <base, loop i>
-  | square i j ⇒ refl
```
Comment (cangiuli) [inline example/torus.red]: :laughing:

Comment (jonsterling) [reply example/torus.red]: taste it

Comment (jonsterling) [reply example/torus.red]: It's a good test, but it is a little confusing in context.

**Pull request closed without merging**